etc/branding/coverimage.xsl

```diff
@@ -98,7 +98,7 @@
           <xsl:value-of select="opencast:XsltHelper.split(metadata/description, 50, 3, true)" />
         </tspan>
         <tspan class="presentationdate" dy="12%" x="50%">
-          <xsl:value-of select="date:format-date(metadata/start, 'MMMMMMMMMM dd, YYYY, HH:mm')" />
+          <xsl:value-of select="date:format-date(metadata/date, 'MMMMMMMMMM dd, YYYY, HH:mm:ss')" />
```

Comment:
Hi @uzk-rrzk Are you sure that you want to display the seconds here? Seems somewhat over-accurate....

Reply:
This is exact the line that I found in the documentation :-)

Reply:
I'm not saying that it would be wrong, but yes: You can indeed chose so omitting :ss will work.

So the question is not whether the doc provides an example but rather whether you would like to see the seconds or not ;-)

I tend to say the seconds are quite useless, but I don't have a strong opinion on that ;-)

Reply:
I agree that seconds make no sense in this case.